### ISSUE TITLE
bach size question 

### ISSUE THREAD

**Issue by Emma20102** [NVIDIA/DIGITS#1675]
hi 

i tried to vary the number of the bach size using 32,64, 128 and 200 but i have noticed that increasing the number of bach size decrease the accuracy? is their any reason behind this?

regards
Eima

**Comment by Emma20102** [comment]
i am using nvidia digit caffe platform and i use the bach size field on the interface for configuring the training and recording the overall result after each complete training

**Comment by lukeyeager** [comment]
Typically you want to increase the learning rate when you increase the batch size. Exactly how the two parameters are related is complex and often discussed/debated.
